.github/workflows/ci.yml

```diff
@@ -4,18 +4,14 @@ on: [push, workflow_call]
 
 jobs:
   prepare:
-    name: Prepare (${{ matrix.os}}, Node ${{ matrix.node }})
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
```

Comment:
No need to run on multiple node versions for me, it's too much for this repo. Also this would mean running all the expensive tests on each or separating into another workflow.